src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.unit.spec.js

```diff
@@ -206,6 +206,7 @@ describe('VAOS Page: VAFacilityPage', () => {
       expect(screen.baseElement).not.to.contain.text('more location');
     });
 
+    // The expected test behavior is different from production code.
```

Comment:
Can someone help me confirm if my understanding here is correct? If so, I think we should remove this test.

Reply:
@jenniemc since this is directly related to [your change last week](https://github.com/department-of-veterans-affairs/vets-website/pull/29403). Can you help me figure this out? Should we remove this test? Or do we need to modify the reducer.js logic?

Reply:
Good catch @JunTaoLuo . We can keep this test but modify the reducer logic.

Reply:
Reducer updated